tests/Feature/BookTest.php

```diff
@@ -82,7 +83,38 @@ public function test_get_book_by_id_invalid(): void
                     ->assertJson([
                         'message' => "Book with id 3 not found"
                     ]);
+    }
+
+    public function test_get_books_filtered_by_genre(): void
+    {
+        
+            $genre = Genre::factory()->create();
+            
+            Book::factory()->count(3)->create(['genre_id' => $genre->id]);
```

Comment:
To make this test more valuable, you should make a second genre, and a few books assigned to that genre - just to make sure you only get the correct books.

Literally just repeat these two lines a second time and leave everything else as it is